NuRadioReco/detector/response.py

```diff
@@ -380,10 +380,14 @@ def plot(self, ax1=None, show=False, in_dB=True, plt_kwargs={}):
         else:
             return fig, ax
 
-    def get_time_delay(self):
+    def get_time_delay(self, ):
```

Comment:
what does the comma do here?

Reply:
I trust you on resolving this, but maybe I can learn: Can you tell me what is the benefit of introducing the comma is here?